modules/ROOT/pages/specify-data-in-a-process-definition.adoc

```diff
@@ -24,6 +24,11 @@ Best practice:
 * Define a variable at the lowest level possible, using a process variable only when required for the process flow.
 * Use the minimum number of variables possible; having an unnecessarily large set of variables has an impact on performance.
 
+[WARNING]
+====
+Business Object should not be stored in Process or activity data. They can only be stored in Business data and *Form* transient data.
```

Comment:
What are Form transient data ? Did you mean transient data ?

Reply:
Form variable are always transient so I guess I don't need that. 
BO should never be stored in activity's transient variable. 

Maybe this will be clearer, what do you think?: 
Business Object should not be stored in Process or activity variable. They can only be stored in Business data variable and form variable.

Reply:
Yeah, it is a little confusing. I would stick with Business Object = Business variables.

Reply:
ok, I made a new commit with clearer sentence.